packages/tpetra/core/test/HashTable/FixedHashTableTest.cpp

```diff
@@ -933,20 +930,11 @@ namespace { // (anonymous)
 
 #ifdef KOKKOS_ENABLE_CUDA
     {
-      // mfh 14 Jan 2016: Only exercise CudaUVMSpace, not CudaSpace,
-      // because Tpetra (and FixedHashTable in particular) assume UVM.
-      // Testing CudaSpace here results in the following exception:
-      //
-      // p=0: *** Caught standard std::exception of type 'std::runtime_error' :
-      //
-      // Kokkos::CudaSpace::access_error attempt to execute Cuda
-      //   function from non-Cuda space
-      // Traceback functionality not available
-      if (! std::is_same<typename DeviceType::memory_space, Kokkos::CudaUVMSpace>::value) {
-        out << "Testing copy constructor to CudaUVMSpace" << endl;
-        TestCopyCtor<KeyType, ValueType, Kokkos::Device<Kokkos::Cuda, Kokkos::CudaUVMSpace>,
+      if (! std::is_same<typename DeviceType::memory_space, Kokkos::CudaSpace>::value) {
+        out << "Testing copy constructor to CudaSpace" << endl;
+        TestCopyCtor<KeyType, ValueType, Kokkos::Device<Kokkos::Cuda, Kokkos::CudaSpace>,
```

Comment:
With this and the other similar changes, is there a reason to not just use the default memory space for Kokkos::Cuda?  That way we can test with UVM or non-UVM, depending on the build.